packages/migrate/src/Migrate.ts

```diff
@@ -155,6 +155,7 @@ export class Migrate {
       printDownloadProgress: true,
       version: enginesVersion,
       cliVersion: packageJson.version,
+      dataProxy: false,
```

Comment:
We don't allow `--data-proxy` on `prisma migrate` as they are not currently compatible.